src/pages/attend.astro

```diff
@@ -0,0 +1,89 @@
+---
+import GettingThere from "../components/GettingThere.astro";
+import HealthSafetyPolicy from "../components/HealthSafetyPolicy.astro";
+import Hero from "../components/Hero.astro";
+import TextAndImage from "../components/TextAndImage.astro";
+import BaseLayout from "../layouts/BaseLayout.astro";
+import SmallMeeting from "../img/SmallMeeting.png";
+import LargeMeeting from "../img/LargeMeeting.png";
+import MatrixStep from "../img/MatrixStep.png";
+import Buffet from "../img/Buffet.png";
+---
+
+<style>
+    #getting-there {
+        background-color: #1a1a1a;
+    }
+</style>
+<BaseLayout
+    pageTitle="Attend The Matrix Conference"
+    description="Join us for the Matrix Conference from October 15 to 18, 2025. The four day conference will take place in Strasbourg, France. Talks will be streamed live and virtual attendees will be able to ask speakers questions."
+>
+    <Hero
+        title="Attend the conference"
+        description="Join us for the Matrix Conference from October 15 to 18, 2025. The four day conference will take place in Strasbourg, France. Talks will be streamed live and virtual attendees will be able to ask speakers questions."
+        ctaText="Get tickets"
+        ctaHref="/register"
+    />
+    <section id="large-end-user-orgs">
+        <TextAndImage imgSrc={SmallMeeting}>
+            <h2>Why large end-user organisations attend</h2>
+            <p>
+                The Matrix Conference is the best place to learn more about
+                Matrix. Hear direct from Matthew Hodgson and Amandine Le Pape,
+                Matrix co-founders.
```

Comment:
Idk, this is of course true, but I feel like the point is rather to hear about all the developments from all the experts or something, not only Matthew's keynote.

Reply:
Very good point. Moving to #109, text suggestions would be super helpdul